pkg/tests/ossm-federation/testdata/failover/east-mesh/smcp.yaml

```diff
@@ -1,25 +1,24 @@
 apiVersion: maistra.io/v2
 kind: ServiceMeshControlPlane
 metadata:
-  name: fed-import
+  name: east-mesh
```

Comment:
Need to add ThirdParty in security section in this script. I know that in 2.3 branch we have a if with variable .Rosa to setup this. We are always test failover in ROSA

Reply:
Ignore this, single cluster needs the variable in ROSA is not needed